lib/adapters/idb/idb.js

```diff
@@ -46,14 +47,10 @@ function IdbPouch(opts, callback) {
 
 function init(api, opts, callback) {
 
-  var name = opts.name;
+  var dbName = opts.name;
```

Comment:
^ this generic name `name` was annoying me, especially because I frequently screw up and accidentally use the global `window.name`, causing hard-to-debug bugs.